tests/plugins/tableselection/manual/draganddropimage.html

```diff
@@ -0,0 +1,42 @@
+<textarea id="editor">
+	<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
+		<tbody>
+			<tr>
+				<td><img alt="" src="https://via.placeholder.com/100" />&nbsp;</td>
+				<td>test</td>
+			</tr>
+			<tr>
+				<td>
+				<p>&nbsp;</p>
+				<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
+					<tbody>
+						<tr>
+							<td>&nbsp;</td>
+							<td>&nbsp;</td>
+						</tr>
+					</tbody>
+				</table>
+
+				<p>&nbsp;</p>
+				</td>
+				<td>&nbsp;</td>
+			</tr>
+			<tr>
+				<td>&nbsp;</td>
+				<td>&nbsp;</td>
+			</tr>
+		</tbody>
+	</table>
+</textarea>
+
+<script>
+	if ( bender.tools.env.mobile ) {
+		bender.ignore();
```

Comment:
Please add `return` statement and wrap the entire code in IIFE. The same for the other manual test.